samples/Erlang/sample.app.src

```diff
@@ -0,0 +1,8 @@
+{application, sample,
```

Comment:
*.app.src is a declarative Erlang data structure, rather than an executable module like *.erl.

[rebar.config](https://github.com/github/linguist/blob/master/samples/Erlang/filenames/rebar.config) is similar example.